GetStream/stream-chat-android
# [3508] Add additional padding to InputField

### 🎯 Goal

Add additional padding to `InputField`.

### 🛠 Implementation details

Split `innerPadding` of the `InputField` to `verticalPadding` and `horizontalPadding`.

### 🎨 UI Changes

| Before | After |
| --- | --- |
| ![Screenshot_20220525_163311](https://user-images.githubusercontent.com/38032787/170287898-ed1c9463-6d98-426b-85dd-c6319198f8d7.png) | ![Screenshot_20220525_163437](https://user-images.githubusercontent.com/38032787/170287926-63151f39-c078-4c1c-904f-d448c4b18512.png) |

### 🧪 Testing

1. Run on develop and check the padding
2. Run on this branch and check the padding

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

![giphy](https://user-images.githubusercontent.com/38032787/170288464-33e23572-b899-443d-b187-0ca4b4aa7317.gif)
